.tool-versions

```diff
@@ -1 +1 @@
-ruby 2.2.7
+ruby 2.7
```

Comment:
We wanted this gem to support 2.2.7 ruby as well. Which means we need to run tests on 2.2.7 version also. I suggest undo this.